src/main/java/uk/gov/hmcts/ccd/appinsights/CallbackTelemetryContext.java

```diff
@@ -0,0 +1,14 @@
+package uk.gov.hmcts.ccd.appinsights;
+
+public class CallbackTelemetryContext {
```

Comment:
Maybe let's call it `TelemetryContext` so that in the future we can use it outside of callbacks as well?

Reply:
I would prefer separate object for each specific purpose. We can always revisit structure/relationship when more use cases come in..